float32[] joint_points
float32   max_vel_fact
float32   max_acc_fact
---
int32 reply
